app/admin/members.rb

```diff
@@ -18,16 +18,18 @@
                 :register_id,
                 group_ids: []
 
+  decorate_with MemberDecorator
+
   index do
     selectable_column
     column :first_name
     column :last_name
     column :role
-    column('3 heures faites?') { |member| member.worked_three_hours?(Date.current) }
     column(:group) do |member|
       group_links = member.groups.map { |group| auto_link group }
       safe_join group_links, ', '
     end
+    column :worked_hours_of_three_last_months
```

Comment:
The I18n translation for this column is missing in the BO. You could either add it as an activerecord virtual attribute in the appropriate locale file, or use an admin view locale lookup (e.g. `column(t '.worked_hours)`)

Reply:
done